frontend/src/components/taskList/taskListRow.tsx

```diff
@@ -302,21 +330,35 @@ export default function TableRowComponent(props: any) {
   }, []);
 
   useEffect(() => {
-    if (!isLoading && buttonState.personId === "") {
+    if (!isLoading && !errors?.title && buttonState.personId === "") {
       TaskService.editTask({
         _id: props._id,
         title: editState.taskName,
         description: editDescriptionState.description,
         status: buttonState.statusText,
-      });
-    } else if (!isLoading) {
+      }).then(
+        () => {
+          enqueueSnackbar("Edytowano task!");
```

Comment:
zadanie? ale w sumie date picker też ma dni po angielskiemu to nwm ;P
i tak samo niżej

Reply:
Zadanie lepiej, zmieniłam